tests/unit/test_charm.py

```diff
@@ -167,7 +165,58 @@ def test_configure_workload_container(
         get_service.assert_called_once()
         stop.assert_called_once()
         start.assert_called_once()
-        self.assertEqual(harness.charm.unit.status, ActiveStatus("Ready"))
+        assert harness.charm.unit.status, ActiveStatus("Ready")
+
+    @mock.patch("ops.model.Container.pull")
+    @pytest.mark.parametrize(
+        "test_input,expected",
+        [
+            ("", []),
+            (
+                f"10.10.10.11 - - [{DATE_NOW}\n10.10.10.11 - - [{DATE_NOW}\n10.10.10.11 - - [{DATE_NOW}\n10.10.10.12 - - [{DATE_NOW}\n10.10.10.12 - - [{DATE_NOW}",  # noqa E501
+                [("10.10.10.11", 3), ("10.10.10.12", 2)],
+            ),
+            (
+                f"10.10.10.11 - - [{DATE_NOW}\n10.10.10.11 - - [{DATE_NOW}\n10.10.10.11 - - [{DATE_NOW}",  # noqa E501
```

Comment:
Same comment as above, consider breaking this up. black should do this for you if you just put `" f"` where you want the breaks to be